setup.py

```diff
@@ -35,6 +35,7 @@
         "Django>=3.2,<4.0",
         "django-cors-headers",
         "django-filer",
+        "django-linkcheck",
```

Comment:
You need to execute `pipenv install -e .` locally and add the changes to your local `Pipfile.lock` to the repository as described in the [documentation](https://integreat.github.io/integreat-cms/virtualenv.html#add-dependencies). No need to change anything to the circleci workflow.